lib/pliny/commands/updater.rb

```diff
@@ -67,8 +67,13 @@ def save_patch(curr, target)
     end
 
     def exec_patch
+      msg = [
+        "Pliny update applied. Please review the changes staged for",
+        "commit, and consider applying the diff in .rej files manually.",
+        "You can then remove these files with `git clean -f`.",
+      ].join("\n")
       # throw .orig and .rej files in /tmp, they're useless with source control
```

Comment:
I think this comment can go now that the `patch ...` line got replaced by `git apply ...` :+1: